docs/feature-gates.md

```diff
@@ -250,8 +245,24 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit.
+Node may lose network connection when `antrea-agent`or OVS daemons are stopped
+unexpectedly, which can be recovered by rebooting the Node.
+`AntreaIPAM` Pods' traffic will not be routed by Node network stack.
+All traffic's destination MAC will be overwritten to Pod's MAC address when the
+destination IP is Pod IP on local Node and will be sent to destination OVS port directly,
+including `AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods, even they are not in
+the same subnets.
+
 #### Requirements for this Feature
 
-This feature is currently supported on Linux Nodes only. Annotation of single IP pool
-is supported. In future, annotation of up to two pools of different IP versions will be
-supported.
+In Antrea 1.4, this feature is supported on Linux Nodes, with IPv4, `system` OVS datapath
+type, and `noEncap`, `noSNAT` traffic mode.
+The `ipRanges` must be in the same subnet as `Node`. This is because `AntreaIPAM` Pods
+use the same L2 network of `Node`. Only a single IP pool can be included in the Namespace
```

Comment:
because cross-Node traffic of AntreaIPAM Pods is sent to the Node network from the source Node interface, and the Node network must be able to forward the traffic to the destination Node

Reply:
Fixed.